.buildkite/scripts/exhaustive-tests/generate-steps.py

```diff
@@ -38,7 +38,7 @@ def compat_linux_step(imagesuffix: str) -> dict[str, typing.Any]:
 
 
 def compat_windows_step(imagesuffix: str) -> dict[str, typing.Any]:
-    windows_command = LiteralScalarString(r'''$$env:WORKSPACE=$$PWD.Path ; .\\ci\\unit_tests.bat''')
+    windows_command = LiteralScalarString(r'''.\\ci\\unit_tests.ps1''')
```

Comment:
Should we drop the env variables here? maybe just just $$PWD.Path?

Reply:
It's deliberate that we drop `WORSPACE` because this old env variable was needed by the old bat script.
Regarding using a full path, when BK checks out a repo it `cd`s under the checked out repo, so it's safe to just invoke scripts relatively. We do the same elsewhere in Linux etc.